### PR TITLE
LPS-64241

### PR DESCRIPTION
**Pull request by jonmak08**
Hey @ealonso,

Attached is an update for http://issues.liferay.com/browse/LPS-64241.  I have sent this to you for review because this was originally changed with Git SHA ID: dc2e6f254a29457e5cf2c6c4fdeb22b72e5cf2e0

Please let me know if you have any questions. Thanks!

### CODE REVIEW

File: modules/apps/web-experience/layout/layout-admin-web/src/main/resources/META-INF/resources/layout/details.jsp [hunk 1]
```diff
@@ -271,7 +271,7 @@ StringBuilder friendlyURLBase = new StringBuilder();
 
 			$('.layout-prototype-info-message').toggleClass('hide', !layoutPrototypeLinkChecked);
 
-			var propagatableFields = $('#<portlet:namespace />fm_editLayout .propagatable-field');
+			var propagatableFields = $('#<portlet:namespace />editLayoutFM .propagatable-field');
```
Comment (blzaugg) [inline modules/apps/web-experience/layout/layout-admin-web/src/main/resources/META-INF/resources/layout/details.jsp]: `editLayoutFM` vs `editLayoutFm`

Comment (jonmak08) [inline modules/apps/web-experience/layout/layout-admin-web/src/main/resources/META-INF/resources/layout/details.jsp]: My fault @blzaugg.  I just pushed up a fix for that.

**Comment by jonmak08** [comment]
ci:retest

**Comment by ealonso** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by ealonso** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/38019. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by ealonso** [comment]
Pull request submitted to https://github.com/liferay-echo/liferay-portal/pull/693. [See changes here.](https://github.com/ealonso/liferay-portal/compare/5ddcb12c85db~2...5ddcb12c85db) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**